src/services/openklant2/types.ts

```diff
@@ -203,5 +205,18 @@ export type Partij = {
   };
 };
 
+export type identificator = {
```

Comment:
Zullen we de conventie aanhouden om een type / interface met een hoofdletter te laten beginnen? Misschien ook iets voor in de eslint configuratie.

Reply:
cd59966b22fca35559d8c8d9bc6e3b9e5d1501bd